src/web/core/src/main/java/org/geoserver/web/MenuPageInfo.java

```diff
@@ -22,7 +22,7 @@
  * @author David Winslow <dwinslow@opengeo.org>
  */
 @SuppressWarnings("serial")
-public class MenuPageInfo extends ComponentInfo<GeoServerBasePage> implements Comparable<MenuPageInfo> {
+public class MenuPageInfo<T extends GeoServerBasePage> extends ComponentInfo<T> implements Comparable<MenuPageInfo> {
```

Comment:
ServiceMenuPageInfo below ` extends MenuPageInfo<T>`. Should we also be using `implements  Comparable<MenuPageInfo<T>>` here?